src/PKSim.CLI.Core/Services/ExportSimulationRunner.cs

```diff
@@ -116,6 +116,12 @@ public async Task<SimulationExport> ExportSimulation(Simulation simulation, Expo
          _lazyLoadTask.Load(simulation);
          _lazyLoadTask.LoadResults(simulation);
 
+         var simulationExportOptions = new SimulationExportOptions
```

Comment:
I have created a small object structure here instead to pass parmaeters. Much nicer to work with